src/areafix.c

```diff
@@ -1315,7 +1317,7 @@ char * subscribe(s_link * link, char * cmd)
                 else
                 {
                     xscatprintf(&report, " %s %s  already linked\r", an,
-                                print_ch(49 - strlen(an), '.'));
+                                repeat_char(49 - strlen(an), '.'));
```

Comment:
```
#include <stdio.h>
#include <string.h>

char * repeat_char(size_t len, char ch)
{   
    printf("len=%zu\n", len);
    return NULL;
}

int main()
{   
    char an[] = "superlongstring_superlongstring_superlongstring_superlongstring";
    repeat_char(49 - strlen(an), '.');
    return 0;
}
```

If you compile and run it, you will get ```len=18446744073709551602```

Reply:
Fortunately there's a sanity check for `len` in the actual `repeat_char()` where it should just return an empty string.